interpreter/CppInterOp/README.md

```diff
@@ -0,0 +1,372 @@
+<div align="center">
+
+# CppInterOp
```

Comment:
As discussed offline, we probably want a similar workflow as `.github/workflows/llvm-diff.yml` to make sure our copies are not diverging